recipe/meta.yaml

```diff
@@ -57,6 +58,8 @@ requirements:
     - python
 
 test:
```

Comment:
Please update `test `to:
```
test:
  requires:
    - pip
    - pytest
  commands:
    - pip check
  imports:
    - Crypto
    - Crypto.Cipher
    - Crypto.Hash
    - Crypto.Protocol
    - Crypto.PublicKey
    - Crypto.Random
    - Crypto.Random.Fortuna
    - Crypto.Random.OSRNG
    - Crypto.SelfTest
    - Crypto.SelfTest.Cipher
    - Crypto.SelfTest.Hash
    - Crypto.SelfTest.Protocol
    - Crypto.SelfTest.PublicKey
    - Crypto.SelfTest.Random
    - Crypto.SelfTest.Random.Fortuna
    - Crypto.SelfTest.Random.OSRNG
    - Crypto.SelfTest.Signature
    - Crypto.SelfTest.Util
    - Crypto.Signature
    - Crypto.Util
```

Reply:
the import validations are there earlier, will enhance the pip check.